docs/css-customization.md

```diff
@@ -1284,3 +1300,171 @@ The frame that adds the "card styles" such as background-color or box-shadow.
+    background-color: #f5f5f5;
+    padding: 10px;
+    border-bottom: 1px solid #ddd;
+}
+```
+
+- _webchat-modal-title_  
+  The title text in modal headers.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-title {
+    font-size: 18px;
+    color: #333;
+    font-weight: bold;
+}
+```
+
+- _webchat-modal-close-button_  
+  The close button in modal headers.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-close-button {
+    background-color: transparent;
+    border: none;
+    color: #666;
+}
+```
+
+- _webchat-modal-close-icon_  
+  The icon within the modal close button.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-close-icon {
+    font-size: 20px;
+    color: #666;
+}
+```
+
+- _webchat-modal-divider-wrapper_  
+  The wrapper for modal dividers.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-divider-wrapper {
+    padding: 10px 0;
+}
+```
+
+- _webchat-modal-divider_  
+  The divider line in modals.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-divider {
+    border-top: 1px solid #ddd;
+}
+```
+
+- _webchat-modal-body_  
+  The main content area of modals.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-body {
+    padding: 15px;
+    background-color: white;
+}
+```
+
+- _webchat-modal-footer_  
+  The footer section of modals.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-modal-footer {
+    padding: 10px;
+    background-color: #f5f5f5;
+    border-top: 1px solid #ddd;
+}
+```
+
+- _webchat-delete-conversation-title_  
+  The title in delete conversation dialog.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-delete-conversation-title {
+    font-size: 16px;
+    font-weight: bold;
+    color: #333;
+}
+```
+
+- _webchat-delete-conversation-button_  
+  The delete button style inside the chat options screen.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-delete-conversation-button {
+    padding: 8px 16px;
+    margin: 5px;
+    border-radius: 4px;
+}
+```
+
+- _webchat-delete-conversation-cancel-button_  
+  The cancel button in delete dialog.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-delete-conversation-cancel-button {
+    background-color: #f5f5f5;
+    color: #333;
+}
+```
+
+- _webchat-delete-conversation-confirm-button_  
+  The confirm button in delete dialog.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-delete-conversation-confirm-button {
+    background-color: #dc3545;
+    color: white;
+}
+```
+
+- _webchat-delete-conversation-text_  
+  The warning text in delete conversation dialog.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-delete-conversation-text {
+    color: #666;
+    margin: 10px 0;
+}
+```
+
+- _webchat-delete-all-conversation-text_  
+  The warning text for deleting all conversations.
+
+```CSS
+[data-cognigy-webchat-root] .webchat-delete-all-conversation-text {
+    color: #dc3545;
+    font-weight: bold;
+}
+```
+
+- _webchat-header-delete-all-conversations-button_  
+  The button to delete all conversations in the header.
```

Comment:
Can please rephrase so that it is clear that you are talking about the previous conversations header